virttest/virt_vm.py

```diff
@@ -638,6 +638,7 @@ def get_address(self, index=0):
         :raise VMAddressVerificationError: If the MAC-IP address mapping cannot
                 be verified (using arping)
         """
+
         nic = self.virtnet[index]
         self.ip_version = self.params.get("ip_version", "ipv4").lower()
```

Comment:
@will-Do I was doing so in my initial version but based on your suggestion as below have updated this PR, now am confused :-(

will-Do 24 days ago Collaborator
@sathnaga
According to this solution, my suggestion is make flexible_index as the 3rd param of this function:

     def get_address(self, index=0, flexible_index=False):
And use index=0 as high priority